### PR TITLE
Remove Rakefile & Gemfile

### PR DESCRIPTION
**Pull request by scannillo**
### Summary of changes

- Remove no-longer-used `Rakefile` & `Gemfile`s
 - Closes out this dependabot PR - #30 

 ### Checklist

 - ~Added a changelog entry~ 

### Authors
@scannillo

### CODE REVIEW

**Comment by scannillo** [comment]
I think we can ignore UI tests failing

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ This document outlines development practices that we follow internally while dev
...
 ## Tests
 
-There are a number of test targets for each section of the project. You can run all tests on the command line with `bundle && rake spec:all`. 
+There are a number of test targets for each section of the project. You can run the following test schemes via Xcode:
 
-It's a good idea to run `rake`, which runs all unit tests, before committing.
-
-The integration tests require a full Braintree stack running on localhost.
+- `UITests`
+- `IntegrationTest`
```
Comment (sarahkoop) [inline DEVELOPMENT.md]: Confirming this one is singular not `IntegrationTests`?

**Review by sarahkoop — APPROVED** [review]

**Review by sshropshire — APPROVED** [review]

File: DEVELOPMENT.md (hunk 1 above)
Comment (scannillo) [reply DEVELOPMENT.md]: You are right, good catch!

Comment (scannillo) [inline DEVELOPMENT.md]:
```suggestion
- `IntegrationTests`
```

**Pull request merged**